subsys/mgmt/mcumgr/smp_bt.c

```diff
@@ -103,48 +104,78 @@ static struct conn_param_data *get_conn_param_data(const struct bt_conn *conn)
 }
 
 /* Sets connection parameters for a given conn. */
-static void conn_params_set(struct bt_conn *conn, bool smp_params)
+static bool conn_params_set(struct bt_conn *conn, struct bt_le_conn_param *params)
```

Comment:
Maybe we could avoid modifying this function signature? I think code used to be simpler with the previous one.

Reply:
In case it automatically submits error work, there may be no need to return anything, because it handles error on its own.